Documentation/ceph-upgrade.md

```diff
@@ -48,116 +49,33 @@ With this upgrade guide, there are a few notes to consider:
-
-1. Make sure the `lvm2` package is installed on the host where OSDs are running. If not, the prepare job will fail
-   and the upgrade of OSDs will be blocked.
-   * This only applies to non-PVC OSDs. Beginning in v1.3, OSDs created on PVCs no longer rely on LVM.
-2. CSI Snapshots: See the next section if you are currently using Ceph-CSI snapshots.
-
-
-### CSI Snapshots
-
-CSI snapshots have moved from Alpha to Beta and are **not** backward compatible. The snapshots
-created with the Alpha version must be deleted before the upgrade.
-
-If you desire to continue using Ceph-CSI 2.x with Alpha snapshots, then:
-
-* Skip this section
-* Set the CSI version to 2.x as described below in the [CSI Version](#csi-version) section.
-
-To continue with the Ceph-CSI v3.0 driver that is recommended with v1.4:
-
-1. List all the volumesnapshots created
-
-```bash
-kubectl get volumesnapshot
-NAME               AGE
-rbd-pvc-snapshot   22s
-```
-
-2. Delete all volumesnapshots
-
-```bash
-kubectl delete volumesnapshot rbd-pvc-snapshot
-volumesnapshot.snapshot.storage.k8s.io "rbd-pvc-snapshot" deleted
-```
-
-3. List all volumesnapshotclasses created
-
-```bash
-kubectl get volumesnapshotclass
-NAME                      AGE
-csi-rbdplugin-snapclass   86s
-```
-
-4. Delete all volumesnapshotclasses
-
-```bash
-kubectl delete volumesnapshotclass csi-rbdplugin-snapclass
-volumesnapshotclass.snapshot.storage.k8s.io "csi-rbdplugin-snapclass" deleted
-```
-
-*Note:* The underlying snapshots on the storage system will be deleted by ceph-csi
-
-#### Delete the Alpha CRDs
-
-As we are updating the snapshot resources from `Alpha` to `Beta` we need to
-delete the old `alphav1` snapshot CRD created by external-snapshotter sidecar container
-
-Check if we have any `v1alpha1` CRD created in our kubernetes cluster
-
-```console
-kubectl get crd volumesnapshotclasses.snapshot.storage.k8s.io -o yaml |grep v1alpha1
-  - name: v1alpha1
-  - v1alpha1
-kubectl get crd volumesnapshotcontents.snapshot.storage.k8s.io -o yaml |grep v1alpha1
-  - name: v1alpha1
-  - v1alpha1
-kubectl get crd volumesnapshots.snapshot.storage.k8s.io -o yaml |grep v1alpha1
-  - name: v1alpha1
-  - v1alpha1
-```
-
-As we have `v1alpha1` CRD created in our kubernetes cluster, we need to delete
-the Alpha CRD
-
-```console
-kubectl delete crd volumesnapshotclasses.snapshot.storage.k8s.io volumesnapshotcontents.snapshot.storage.k8s.io volumesnapshots.snapshot.storage.k8s.io
-
-customresourcedefinition.apiextensions.k8s.io "volumesnapshotclasses.snapshot.storage.k8s.io" deleted
-customresourcedefinition.apiextensions.k8s.io "volumesnapshotcontents.snapshot.storage.k8s.io" deleted
-customresourcedefinition.apiextensions.k8s.io "volumesnapshots.snapshot.storage.k8s.io" deleted
-```
-
-Finally, if you desire to use the beta snapshots, check that the [prerequisites](ceph-csi-snapshot.md#prerequisites) are met.
-
+TODO: are there breaking changes?
```

Comment:
There are a few minor considerations, mentioned in the breaking changes in the pending release notes, but I don't think they are critical enough to mention in the upgrade guide. How about if we add a link to the release notes so they can review the breaking changes?